docs/spec/slashing/state-machine.md

```diff
@@ -0,0 +1,174 @@
+## Transaction & State Machine Interaction Overview
```

Comment:
Should probably be an `h1` heading and and all subsequent headers increased but not a big deal.

Reply:
Sure; updated & also updated in `state.md`.